packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -313,7 +314,9 @@ export class CommandsTreeViewProvider implements vscode.TreeDataProvider<TreeVie
         continue;
       }
 
-      const disposable = vscode.commands.registerCommand(treeItem.commandId, treeItem.callback!);
+      const disposable = vscode.commands.registerCommand(treeItem.commandId, (...args) =>
+        Correlator.run(treeItem.callback!, args)
```

Comment:
1. why need correlator here? 
2. looks like the add function is not used any more?

Reply:
1. The correlator here is for adding correlation id for sign in
2. sign in is still using this API to register command